package.json

```diff
@@ -42,7 +42,6 @@
     "eslint-plugin-react": "7.28.0",
     "eslint-plugin-storybook": "^0.5.6",
     "file-loader": "^6.2.0",
-    "popper.js": "^1.16.1",
```

Comment:
The commit message doesn't explain why this is being removed.  I'm sure it's obvious to you, but it would be helpful to reviewers to explain it in the commit. :)

Reply:
https://getbootstrap.com/docs/5.0/migration/ Bootstrap now uses Popper v2.x. Hence, this dependency was causing conflicts.

Reply:
Thanks, but it should be added to the commit message too. :)